### PR TITLE
core[minor]: Add get_input_jsonschema, get_output_jsonschema, get_config_jsonschema

### PR DESCRIPTION
**Pull request by eyurtsev**
This PR adds methods to directly get the json schema for inputs, outputs, and config.
Currently, it's delegating to the underlying pydantic implementation, but this may be changed in the future to be independent of pydantic.

### CODE REVIEW

**Review by nfcampos — APPROVED** [review]

File: libs/core/langchain_core/runnables/base.py [hunk 1]
```diff
@@ -350,6 +350,34 @@ def get_input_schema(
...
+                def add_one(x: int) -> int:
+                    return x + 1
+
+                runnable = RunnableLambda(add_one)
+
+                print(runnable.get_input_jsonschema())
+
+        .. versionadded:: 0.3.0
+        """
+        return self.get_input_schema(config).schema()
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/base.py:379]:
```suggestion
        return self.get_input_schema(config).model_json_schema()
```

File: libs/core/langchain_core/runnables/base.py [hunk 2]
```diff
@@ -382,6 +410,34 @@ def get_output_schema(
...
+                def add_one(x: int) -> int:
+                    return x + 1
+
+                runnable = RunnableLambda(add_one)
+
+                print(runnable.get_output_jsonschema())
+
+        .. versionadded:: 0.3.0
+        """
+        return self.get_output_schema(config).schema()
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/base.py:439]:
```suggestion
        return self.get_output_schema(config).model_json_schema()
```

File: libs/core/langchain_core/runnables/base.py [hunk 3]
```diff
@@ -435,6 +491,21 @@ def config_schema(
...
+
+        Args:
+            include: A list of fields to include in the config schema.
+
+        Returns:
+            A JSON schema that represents the output of the Runnable.
+
+        .. versionadded:: 0.3.0
+        """
+        return self.config_schema(include=include).schema()
```
Comment (eyurtsev) [inline libs/core/langchain_core/runnables/base.py:507]:
```suggestion
        return self.config_schema(include=include).model_json_schema()
```

**Comment by eyurtsev** [comment]
Force merging this one. It looks like CI is failing to avoid testing with 3.8 since the original PR was using an older version of the CI set up

**Pull request merged**